### PR TITLE
feat/add helpers

### PR DESCRIPTION
**Pull request by mvarlic**
* add helpers

### CODE REVIEW

File: src/util/Constans.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export default class Constans {
```
Comment (Matiasnickolas) [inline src/util/Constans.js]: typo here: should be Constants

File: src/util/Constans.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+    static MONTHS_OF_YEAR = ['Enero', 'Febrero', 'Marzo', 
+        'Abril', 'Mayo', 'Junio', 
+        'Julio', 'Agosto', 'Setiempre',
```
Comment (Matiasnickolas) [inline src/util/Constans.js]: typo: Septiembre

File: src/util/TabConstans.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+export default class TabConstans {
```
Comment (Matiasnickolas) [inline src/util/TabConstans.js]: typo: TabConstants

File: src/util/TabConstans.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    static DASHBOARD = 0;
+    static WEBPAYPLUS = 1;
+    static WEBPAYPLUS_MALL = 2;
+    static ONECLICK = 3;
+    static TRANSACTION = 4;
+    static INSCRIPTION = 5;
+    static API_SERVICE_LOG = 6;
+    static EXECUTION_ERROR_LOG = 7;
+    static DIAGNOSTICO = 8;
```
Comment (Matiasnickolas) [inline src/util/TabConstans.js]: Should this be in English?

File: src/util/GridFilterTemplateUtil.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
+import { Dropdown } from "primereact/dropdown";
+import { ENVIROMENT_LIST, INSCRIPTIONS_STATUS, PRODUCT_LIST, SERVICE_LIST, 
+    TRANSACTION_STATUS_LIST, TRANSACTION_TBK_REFUND_LIST, TRANSACTION_TBK_STATUS_LIST } from "./TbkConstans";
+
+export const createFilerDropdown = (list) => {
```
Comment (Matiasnickolas) [inline src/util/GridFilterTemplateUtil.js:5]: this should be createFilterDropdown?

File: src/util/Constans.js (hunk 1 above)
Comment (mvarlic) [reply src/util/Constans.js]: fixed

File: src/util/TabConstans.js (hunk 3 above)
Comment (mvarlic) [reply src/util/TabConstans.js]: fixed

File: src/util/TabConstans.js (hunk 4 above)
Comment (mvarlic) [reply src/util/TabConstans.js]: fixed

File: src/util/Constans.js (hunk 2 above)
Comment (mvarlic) [reply src/util/Constans.js]: fixed

File: src/util/GridFilterTemplateUtil.js (hunk 5 above)
Comment (mvarlic) [reply src/util/GridFilterTemplateUtil.js:5]: It is only for grid columns with filters

**Pull request closed without merging**